db.js

```diff
@@ -515,15 +509,17 @@ var users = (function () {
       });
     },
     insert: function (user) {
-      var sql = "INSERT INTO users (name, google_id, picture) VALUES (?,?,?)";
-      var inserts = [user.name, user.google_id, user.picture];
+      var sql =
+        "INSERT INTO users (name, google_id, picture, email) VALUES (?,?,?,?)";
+      var inserts = [user.name, user.google_id, user.picture, user.email];
```

Comment:
If you update a db function, you have to go find all the instances where it's used. I think for this one, you have to send in an empty string for email address in auth.js when creating a new user.